hybrasyl/Objects/User.cs

```diff
@@ -2006,7 +2006,7 @@ public bool RemoveItem(string itemName, byte quantity = 0x01, bool updateWeight
             if (Inventory.Contains(itemName, quantity))
             {
                 var remaining = (int)quantity;
-                var slots = Inventory.SlotOf(itemName);
+                var slots = Inventory.SlotByName(itemName);

```

Comment:
I don't think this takes into account that you can have multiple items of the same name in many slots. Would need to verify though

Reply:
Underlying implementation didn't change, just signature

Reply:
Was more of a note to keep in mind, as I believe it was on the todo list way back when